docs/topics/web-create-project.adoc

```diff
@@ -26,7 +26,9 @@ Registering the server path of an application ensures that {ProductShortName} al
 +
 image::web-configure-analysis.png[Transformation targets]
 . Click *Next*.
-. Select packages and click *>* to include them in the analysis.
+. Package selection
++
```

Comment:
@sbeskin-redhat details about the default behaviour and the toggle switch are missing.

Quoting from the JIRA:
The package selection screen show the packages on the left hand side that are known libraries and will, by default, be excluded from the analysis

To fine tune the package selection criteria, **toggle the switch** above the Included packages panel to On and use the arrow buttons to manage which packages will be included in the analysis.

Reply:
@stillalearner The toggle switch is indeed mentioned in jira, but on the screenshot that was attached later, there is no switch! 
And the way it looks on the screen, all available packages are in the left-side pane, not only the known libraries.

Reply:
@stillalearner  Oh. Maybe you can send me the latest screen?

Reply:
> @sbeskin-redhat I think the screenshot is not correct then, it looks like a screenshot from MTR 1.0.1, instead of 1.1.0. Better to confirm it once from @mrizzi

I do confirm: the screenshot was taken from MTR 1.0.1 so it can not work with MTR 1.1.0's doc.
To have the updated screenshot it will take some time since it will be available onle once MTR 1.1.0 Web Console will be staged for QE.